jobs/deis-v2-e2e.groovy

```diff
@@ -0,0 +1,104 @@
+evaluate(new File("./common.groovy"))
+
+def getProjectChannel () {
+  repos.each { Map repo ->
+    // TODO: ? How to inspect jenkins environment to perform check below ?
+    if ("${repo.name.toUpperCase().replaceAll('-', '_')}_SHA" != '') {
```

Comment:
@sgoings ^^ this not quite right.  Possible to inspect Jenkins env for particular value?